includes/class-wp-notify-demo.php

```diff
@@ -0,0 +1,249 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
Can I ask why this file has been renamed?

It's functional code rather than a class, so feels inaccurate,

Reply:
Let's say it was the beginning of trasposing "demo" to be a real plugin file, imho it wouldn't hurt to create an init class for the notices plugin. Moreover, actually there is a class for tables inside this file should be isolated.
Do you think it would be better to do it directly in this PR or leave it for when this is accepted? frankly, it would be better if you did this one!

Reply:
I think having it in the class file without fully restructuring is going to cause some problems later tbh. What do you think about renaming it back to demo.php for now, and then I can pick up refactoring the PHP once this is merged if that works for you?